roles/openshift_node_upgrade/tasks/restart.yml

```diff
@@ -22,7 +22,6 @@
     - "{{ openshift.common.service_type }}-master-controllers"
     - "{{ openshift.common.service_type }}-node"
   failed_when: false
-  when: openshift.common.is_containerized | bool
```

Comment:
Cleaning up the task name to say `Start services` as we're not actually restarting but only starting and tolerating services that don't exist or are masked because of HA installs.